R/read.quitte.R

```diff
@@ -129,6 +129,10 @@ read.quitte <- function(file,
             stop("missing default columns in header of file ", f, ": ",
                  paste(setdiff(default.columns, header[1:5]), collapse = ", "))
 
+        if (length(period.columns) == 0) {
+            stop("No column name found that could be understood as a 4-digit year.")
```

Comment:
Please indicate the file which failed to read, as `read.quitte()` can read several files at once.